### ISSUE TITLE
Adding wildcard to identify cookies with dynamic prefix

### ISSUE THREAD

**Issue by fellyph** [GoogleChromeLabs/ps-analysis-tool#424]
## Feature Request

**Description:**
Some platforms use cookies with a dynamic prefix, for example, `[site_id]_l` or `[visitor_id]_tv`. Currently, the extension supports suffix wildcards, like `KRTBCOOKIE_*`. The suggestion for this feature is to apply the same logic as used for dynamic suffixes on prefixes.

**Motivation:**
Implementing this feature will help identify the category of cookies with dynamic prefixes from the same domain.

**User Story:**
As a developer, I want to see the category from cookies with a dynamic prefix so that I can better debug.

**Acceptance Criteria:**
- [ ] be able to see the cookie's categories when handles a dynamic prefix

**Additional Information:**

**Cookies with dynamic suffix**
![Screenshot 2024-01-15 at 11 49 42](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/a2778f56-6350-4a9d-b7b6-86860e6b439d)

**Cookies with dynamic prefix**
![Screenshot 2024-01-15 at 11 51 27](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/b0aaa627-9aa7-4869-8db2-ae0c749ef179)

**Comment by ayushnirwal** [comment]
@fellyph Would you please add an example of a website that uses cookie names with dynamic prefixes?

**Comment by fellyph** [comment]
Hey @ayushnirwal , ´https://elpais.com´ uses it.

**Comment by ayushnirwal** [comment]
While resolving this issue a unit test was added to the utility `findAnalyticsMatch` testing its capability to match with postfix and prefix wildcard entries in CokieDB. It was realized that it was capable of matching for prefix wildcard entries if they were in CookieDB.

The tool is already capable of matching entries like the one shown below if they were in the DB.

```
{
...
"*_lv": [
    {
      "platform": "TEST PLATFORM",
      "category": "Marketing",
      "name": "*_lv",
      "domain": ".newsroom.bi",
      "description": "TEST DESCRIPTION",
      "retention": "14 days",
      "dataController": "TEST CONTROLLER",
      "gdprUrl": "",
      "wildcard": "1"
    }
  ]
...
}
```

The PR #438 will add the unit test above.

**Comment by maitreyie-chavan** [comment]
Addressed as a part of [v0.5](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.5.0)
